ckanext/push_errors/tests/fixtures.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+from unittest.mock import MagicMock, patch
+from ckanext.push_errors.plugin import PushErrorsPlugin
+
+
+@pytest.fixture
+def mock_app():
+    """Simula una aplicación para pruebas."""
+    app = MagicMock()
+    app.register_error_handler = MagicMock()
+    app.logger = MagicMock()
+    return app
+
+
+@pytest.fixture
+def mock_config():
+    """Mockea la configuración de CKAN."""
+    with patch('ckan.plugins.toolkit.config') as config:
+        config.get.side_effect = lambda key, default=None: {
+            'ckanext.push_errors.url': 'http://mock-url.com',
+            'ckan.site_url': 'http://mock-site.com',
+            'ckanext.push_errors.method': 'POST',
+            'ckanext.push_errors.headers': '{"Authorization": "Bearer {site_url}"}',
+            'ckanext.push_errors.data': '{"error": "{message}"}',
+            # No configuramos explícitamente el título, para que use el default_title
+        }.get(key, default)
+        yield config
```

Comment:
CKAN already provides `ckan_config` to redefine config values in tests

Example:
```py
@pytest.mark.ckan_config("ckan.resource_proxy.max_file_size", 10)
    @pytest.mark.ckan_config("ckan.plugins", "resource_proxy")
    @pytest.mark.usefixtures("with_plugins")
    def test_ignore_declared(self, command):
        result = command("undeclared", "-idatapusher", "-idatastore")
        assert not result.output
        assert not result.exit_code, result.output
```

This is probably non required

Reply:
?